docs/roles/add-role.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Add a Role
+intro: Add roles to organize permissions that can be re-used on environments team memberships.
+links:
+    overview: Project owners can create roles to be used when managing environments team permissions.
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add a Role
+1. On Devopness upper-left corner, click the Devopness logo to see a list of existing projects
+1. Select a `Project`
+1. On the upper-right click `ROLES` to see a list of existing `Roles`
+1. On the upper-right corner of the list click `ADD ROLE`
+1. Provide a name to the `Role`
+1. Optionally type a `Description` about thow this role is for or what permissions it grants
+1. Click `NEXT`
+1. In the `Permissions` step select the permissions for each resource type
```

Comment:
```suggestion
1. In the `Permissions` step select the permissions for each resource type
    1. Make sure the `VIEW` permission in the resource type `SERVER` is marked, otherwise the team member will not have direct access to the environment through the web app.
```
It may be helpful until a better solution for this issue is found.